releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,47 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.1/1.2.1 - January 13, 2020
+
+* Added Support for Spring Boot 2.2.x Hoxton with version 1.2.1. Spring Boot 2.1.x Greenwich is still supported with 1.1.1.
+
+### Configuration
+
+* 4 configuration parameters have renamed/replaced/removed. See [starter](https://github.com/microsoft/spring-cloud-azure/blob/master/spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md) for updated descriptions.
+
+```properties
+# The following configurations have been removed
+spring.cloud.azure.appconfiguration.watch.enabled
+spring.cloud.azure.appconfiguration.managed-identity.object-id
+
+# The following property has been renamed
+spring.cloud.azure.appconfiguration.watch.delay -> spring.cloud.azure.appconfiguration.cache.expiration
+
+# Endpoint Requires URI value
+spring.cloud.azure.appconfiguration.stores[0].name -> spring.cloud.azure.appconfiguration.stores[0].endpoint
+```
+
+* Watch enabled has been removed. To control the refresh rate the cache expiration configuration can be used. By default 30s.
+* Managed Identity doesn't need an object id any more.
+* Watch delay was renamed to cache expiration to reflect new use.
+* For clarity the name configuration has been changed to endpoint. Instead of my-configstore-name use `https://my-configstore-name.azconfig.io`
+
+### Authentication
+
+* TokenCredentialProvider has been split into two file AppConfigCredentialProvider and KeyVaultCredentialProvider.
```

Comment:
I don't think we talked about `TokenCredentialProvider` previously, so it's really the new introduction of `AppConfigCredentialProvider` and `KeyVaultCredentialProvider`.